application-flashmessages-ui/src/main/resources/Flash/DisplayFlashMessages.xml

```diff
@@ -275,7 +275,7 @@
 #flashMessages {
   background-color: $theme.menuBackgroundColor;
   box-shadow: 0 2px 8px rgba(0, 0, 0, 0.4) inset;
-  color: #fff;
+  color: $theme.menuLinkColor;
```

Comment:
Question: where is $theme defined (in what module)? I'm asking since depending on where it's located, you may need to add a maven dependency to that module.